metricbeat/module/php_fpm/process/_meta/data.json

```diff
@@ -1,9 +1,25 @@
 {
-    "@timestamp": "2018-09-25T16:47:04.998Z",
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "agent": {
+        "hostname": "host.example.com",
+        "name": "host.example.com"
+    },
+    "event": {
+        "dataset": "php_fpm.process",
+        "duration": 115000,
+        "module": "php_fpm"
+    },
+    "http": {
```

Comment:
@webmat Could you have a look if this is what you have expected? (check the full data.json)

Reply:
Yes, looks pretty good. One comment about method, but that's it